server/src/test/java/org/apache/druid/server/QueryStackTests.java

```diff
@@ -385,4 +374,22 @@ public static JoinableFactory makeJoinableFactoryFromDefault(
 
     return new MapJoinableFactory(setBuilder.build(), mapBuilder.build());
   }
+
+  public static Injector injector()
```

Comment:
can this be merged with `CalciteTestInjectorBuilder`?

Reply:
they live right now in different modules (`sql` / `server`)  ; but I've tried to unify the common parts and in the process I've removed the `CalciteTestInjectorBuilder`  class